### DISCUSSION TITLE
destination-sftp: CSV support on sftp destination connector

### DISCUSSION THREAD

**Discussion by sivankumar86** [airbytehq/airbyte#36134] [New Connector Request]
### Connector Name

destination-sftp

### What type of integration

Destination

### Revelant Information

Want to export data to SFTP server in CSV format .

### Contribute

- [ ] Yes, I want to contribute

**Comment by AlxGlx** [comment]
What would be the preferred solution to this use case? There seem to be 3 options that spring to mind.

- A new SFTP CSV destination (similar to the SFTP JSON destination)
- Altering of the SFTP destination to allow for multiple filetypes (CSV, JSON...)
- Altering the Files destination to allow for remote locations

I think all options are valid, but in the interests of being succinct I would suggest the last one, altering the Files destination, with the default being the /local directory as it is currently, and a Remote Location dropdown to manage the connection to remote host or folder.

[a comment by an automated account is not shown]
